pkg/flare/sysprobe/archive_linux.go

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
```

Comment:
```suggestion
// Copyright 2024-present Datadog, Inc.
```
please change accordingly in all new files

Reply:
Updated, thanks